src/transformers/generation/logits_process.py

```diff
@@ -1064,8 +1064,8 @@ class SequenceBiasLogitsProcessor(LogitsProcessor):
     </Tip>
 
     Args:
-        sequence_bias (`Dict[Tuple[int], float]`):
-            Dictionary that maps a sequence of tokens to its bias term. Positive biases increase the odds of the
+        sequence_bias (`Union[Dict[Tuple[int], float], List[List[Union[List[int], float]]]]`):
```

Comment:
```suggestion
        sequence_bias (`List[List[Union[List[int], float]]]`):
```
let's not document the case that doesn't support serialization, even though we can accept the format :) that way, we reduce the odds of users reaching the serialization issue

Also related to this suggestion: there should be no trace of the previous format in the docstring (including in the example below) and type hints

Reply:
ok, done